examples/ci/app/ci/utils.ts

```diff
@@ -65,14 +65,21 @@ export const getTestConfig = async (route: string) => {
 
 export const initiateTest = async (route: string, waitForSeconds: number) => {
   const randomTestId = nanoid()
-  const { headers, payload, expectedCallCount, expectedResult } = await getTestConfig(route)
+  const { headers, payload, expectedCallCount, expectedResult, shouldWorkflowStart = true } = await getTestConfig(route)
 
   const { messageId } = await qstash.startWorkflow({ route, headers, payload }, randomTestId)
 
   // sleep for 4 secs and check that message is delivered
   await new Promise(r => setTimeout(r, 4000));
 
-  await qstash.checkWorkflowStart(messageId)
+  try {
+    await qstash.checkWorkflowStart(messageId);
```

Comment:
what about the case `shouldWorkflowStart` is `false`, but the workflow starts?

Reply:
I did check this case today, and realised that checkWorkflowStart doesn't work as I expect at all :D

I will remove it altogether in https://github.com/upstash/workflow-js/pull/32 after we merge this. I didn't want to cause conflicts.